backend/prisma/schema.prisma

```diff
@@ -41,3 +42,12 @@ model Comment {
   jobListing  JobListing @relation(fields: [jobListingId], references: [id])
   jobListingId Int
 }
+
+model Story {
+  id         Int      @id @default(autoincrement())
+  userId     Int
```

Comment:
nit: I think author may be a better name here than userId but not a big deal

Reply:
Anyway thank you for storing this relation as userId like we talked about instead of something like the user object itself so we can keep our objects as lightweight as possible and with the least number of dependencies